<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Login_Form</class>
 <widget class="QMainWindow" name="Admin_Login_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Login Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/assets/presence_logo.png</normaloff>:/res/assets/presence_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget #centralwidget {
	background-color: #000F2A;
}</string>
   </property>
   <widget class="QGroupBox" name="appGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="presenceIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/presence_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="appTitleLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>ExtraBold</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 35px;
	font-style: normal;
	font-weight: 800;
	line-height: normal;
}
</string>
     </property>
     <property name="text">
      <string notr="true">resence</string>
     </property>
    </widget>
    <widget class="QLabel" name="appSubtitleLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	font-size: 9px;
	font-style: normal;
	font-weight: 400;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string notr="true">KEEP IN TRACK EVERYTIME</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="formGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>360</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>430</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/form_background.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="adminIdGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>110</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
	background: none
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="adminIdHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {	
	background: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/login_field.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="adminIdTextbox">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>0</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
      </property>
      <property name="maxLength">
       <number>9</number>
      </property>
      <property name="placeholderText">
       <string>Admin No.</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="adminPinGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>160</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
	background: none
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="adminPinHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {	
	background: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/login_field.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="adminPinTextbox">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>0</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Pin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminPinToggle">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>4</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/assets/hide.png</normaloff>:/res/assets/hide.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>330</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/login_button.png</normaloff>:/res/assets/login_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>210</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>310</y>
       <width>310</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background: none;
	color: #FF003D;
	font-family: Poppins;
	font-size: 10px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
